build is broken

```
$ tsc -b
src/commands/export.ts:44:38 - error TS2339: Property 'exportStream' does not exist on type 'API'.

44       const stream = await APIClient.exportStream(args.noteId, exportType)
                                        ~~~~~~~~~~~~

src/commands/export.ts:50:25 - error TS2339: Property 'exportStream' does not exist on type 'API'.

50         await APIClient.exportStream(args.noteId, exportType)
                           ~~~~~~~~~~~~


Found 2 errors.
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   },
   "bugs": "https://github.com/hackmdio/codimd-cli/issues",
   "dependencies": {
-    "@hackmd/api": "^1.0.1",
+    "@hackmd/api": "^1.1.0",
     "@oclif/command": "^1.5.19",
     "@oclif/config": "^1.13.3",
     "@oclif/plugin-help": "^2.2.3",

